projectile/animate-projectile.ipynb

```diff
@@ -62,6 +62,17 @@
     "    return v * np.cos(theta) * t, v * np.sin(theta) * t - (g/2) * t**2"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# COMMENT\n",
+    "\n",
+    "Is there a reason that you moved away from the original version in which velocity is originally treated as a constant? Since you still follow that storyline later (introducing \"new projectile launching technologies\") I think it would help to also start with v as some hard-coded constant. FWIW presenting it that way did appear to resonate when I presented it.\n",
+    "\n",
+    "To be clear, I'm specifically referring to the wording in the below comment cell, as well as the choice to make v a variable instead of hard coding initially to demonstrate how inflexible code can be made more easily parameterizable. At the very least, we could state explicitly here that velocity is a constant at this point."
```

Comment:
Do you mean it should say

> Let's observe the effect of choosing different launch angles:

instead of

> Let's observe the effect of choosing different velocities and launch angles:

?

Reply:
Yes essentially. The initial study should span a one-parameter space, and the text should reflect that.

Reply:
I agree and will update the notebook to reflect that.